src/Engine/Data/Mesh.hpp

```diff
@@ -412,6 +413,101 @@ class RA_ENGINE_API PolyMesh : public IndexedGeometry<Core::Geometry::PolyMesh>
     Core::AlignedStdVector<IndexType> m_triangleIndices;
 };
 
+/// create a TriangleMesh, PolyMesh or other Core::*Mesh from GeometryData
```

Comment:
If this method only deals with coreMesh as the doc said, and as the template parameter, the return type, the name of the method and the type of parameters comes all from Core, why is this method defined in Engine and not in Core ?